app.yaml.ctmpl

```diff
@@ -28,8 +30,13 @@ libraries:
 env_variables:
   # The following values are to be replaced by information from the output of
   # 'gcloud endpoints services deploy swagger.json' command.
-  ENDPOINTS_SERVICE_NAME: YOUR-PROJECT-ID.appspot.com
-  ENDPOINTS_SERVICE_VERSION: 2016-08-01r0
+  ENDPOINTS_SERVICE_NAME: {{$googleProject}}.appspot.com
```

Comment:
you need to define this in the top of the tmpl file

Reply:
add `{{with $googleProj := env "GOOGLE_PROJ"}}` to the top and another `{{end}}` clause to the end.  this expects an env var called `GOOGLE_PROJ` to exist when rendering; see my below comment about rendering for how to pass that through